src/Illuminate/Events/Dispatcher.php

```diff
@@ -578,17 +579,21 @@ protected function queueHandler($class, $method, $arguments)
     {
         [$listener, $job] = $this->createListenerAndJob($class, $method, $arguments);
 
-        $connection = $this->resolveQueue()->connection(method_exists($listener, 'viaConnection')
-                    ? $listener->viaConnection()
-                    : $listener->connection ?? null);
```

Comment:
Please undo all of these formatting changes.

Reply:
Done.

Reply:
These are still present.

Reply:
What are you talking about? There's new code and that's why the formatting looks like it's different.

Reply:
It's not. Lines 582 and 583 have been re-indented, making the PR harder to review for the actual changes. Please undo this. Same for lines 587 and 588.

Reply:
Corrected.